### PR TITLE
Added CRUD operations for the tasks

### PR DESCRIPTION
**Pull request by AbelG101**
## Changes
- Added another script to store the functions for:
    - Adding task 
    - Removing task
    - Updating the task
    - Storing on local storage
- Added new styles for the task container.
- Modified the label to an input to store the tasks.

### CODE REVIEW

File: src/template.html [hunk 1]
```diff
@@ -16,11 +16,10 @@
...
       <div class="item">
-        <input type="text" placeholder="Add to your list...">
+        <input class="task-adder-input" type="text" placeholder="Add to your list...">
       </div>
       <div class="item">
         <div class="tasks-container">
-          
         </div>
       </div>
       <div class="item">
```
Comment (HAMMAS-SALEEM) [inline src/template.html]: todolist (copy)


- [ ] In your design, please insert this add button with functionality to add a new todo in your HTML to follow the design. :blush:

File: src/JS/taskOperations.js [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+
+const removeTask = (index) => {
+  const tasksElt = document.querySelectorAll('.task');
+  tasksList.splice(index, 1);
+  tasksElt[index].remove();
+  for (let i = index + 1; i < tasksList.length; i += 1) {
+    tasksList[i].index -= 1;
+  }
+  localStorage.setItem('tasks: ', JSON.stringify(tasksList));
+};
```
Comment (HAMMAS-SALEEM) [inline src/JS/taskOperations.js]: - [ ] After removing a todo the index number of each todo should reset. This means if you have 5 todos and if you remove 3rd todo then your fourth and fifth todo index number should decrease by 1. I can see you have implemented it but it is not working. Please fix it :+1:

File: src/template.html [hunk 3]
```diff
@@ -16,11 +16,10 @@
         <div class="heading">Today's To Do</div>
       </div>
       <div class="item">
-        <input type="text" placeholder="Add to your list...">
+        <input class="task-adder-input" type="text" placeholder="Add to your list...">
```
Comment (HAMMAS-SALEEM) [inline src/template.html:20]: - [ ] Your form should be empty after adding a todo. Please fix it. :+1:

File: src/JS/taskOperations.js [hunk 4]
```diff
@@ -0,0 +1,92 @@
...
+  tasksList = JSON.parse(localStorage.getItem('tasks: '));
+  if (tasksList !== null) {
+    tasksList.forEach((taskToBeDone) => {
+      addTaskToPage(taskToBeDone);
+    });
+  } else {
+    tasksList = [];
+  }
+};
+
```
Comment (HAMMAS-SALEEM) [inline src/JS/taskOperations.js:45]: ### Your MIT.md file
todolist
- [ ] Please add app type in your MIT.md file. You can add **WEBSITE**  or **WEBAPP**. This is the license file of your project so please take it seriously :thinking:

**Review by HAMMAS-SALEEM — CHANGES_REQUESTED** [review]
# Required Changes :recycle:
almost-there

### Hey @AbelG101 :wave:

### Highlights :1st_place_medal:
 
:heavy_check_mark: Following Linters
:heavy_check_mark: Descriptive PR Title
:heavy_check_mark: Descriptive PR Summary

Good job on implementing some of the requirements, there are some issues that you still need to work on to go to the next project but you are almost there! :blush:

# Optional Suggestions

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding :clap: :clap: :clap:

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @HAMMAS-SALEEM in your question so I can receive the notification.

_**Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.**_

------------

_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/template.html (hunk 1 above)
Comment (AbelG101) [reply src/template.html:25]: @HAMMAS-SALEEM there's no functionality for this button on the video showing the design, so I'm not sure what to do with it.

**Review by muneebulrehman — APPROVED** [review]
Hi @AbelG101 , 

Your project is complete!  There is nothing else to say other than... it's time to merge it 🥳
Congratulations! 🎉 

# Status: Approved✔️

## Highlights
- No linter errors
- Great follow-up with the pervious reviewer
- Project functionality working as required.

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**